core/src/main/java/io/opentelemetry/android/internal/features/persistence/DiskManager.kt

```diff
@@ -54,31 +52,15 @@ internal class DiskManager(
                 return storedSize
             }
             val requestedSize = diskBufferingConfig.maxCacheSize
-            val availableCacheSize =
-                cacheStorage.ensureCacheSpaceAvailable(requestedSize.toLong()).toInt()
-            // Divides the available cache size by 3 (for each signal's folder) and then subtracts the
-            // size of a single file to be aware of a temp file used when reading data back from the
-            // disk.
-            val maxCacheFileSize = maxCacheFileSize
-            val calculatedSize = availableCacheSize / 3 - maxCacheFileSize
-            if (calculatedSize < maxCacheFileSize) {
-                Log.w(
-                    RumConstants.OTEL_RUM_LOG_TAG,
-                    String.format(
-                        "Insufficient folder cache size: %s, it must be at least: %s",
-                        calculatedSize,
-                        maxCacheFileSize,
-                    ),
-                )
-                return 0
-            }
+
+            // Divides the available cache size by 3 (for each signal's folder)
+            val calculatedSize = requestedSize / 3
```

Comment:
Probably we would no longer need to store this calculation since it's not a heavy one anymore.

Reply:
@deividasstr want to pick this up in this PR or the next?

Reply:
I will do it in this one.
Just in case, I noticed that this value here is used by another sdk, [disk buffering](https://github.com/open-telemetry/opentelemetry-java-contrib/blob/main/disk-buffering/README.md). There shouldn't be no negative backwards compatibility consequence to change cache size on existing files, right?

Reply:
It should be fine. The size config number is read before writing to ensure that we won't take more space than needed, and when the used space plus the one that is about to be written goes over said number, disk buffering clears the oldest data before doing the writing. So to your question on what would happen if the cache size changes, if the new value is higher, then the existing oldest data will remain there longer, and if it's shorter, then more older data will be cleared in the next write (unless it gets exported first).